iroh-base/src/key.rs

```diff
@@ -334,40 +334,13 @@ impl SecretKey {
         self.secret.verifying_key().into()
     }
 
-    /// Generate a new [`SecretKey`] with the default randomness generator.
-    pub fn generate() -> Self {
-        let mut rng = rand::rngs::OsRng;
-        Self::generate_with_rng(&mut rng)
-    }
-
     /// Generate a new [`SecretKey`] with a randomness generator.
-    pub fn generate_with_rng<R: CryptoRngCore + ?Sized>(csprng: &mut R) -> Self {
-        let secret = SigningKey::generate(csprng);
+    pub fn generate<R: CryptoRngCore>(mut csprng: R) -> Self {
+        let secret = SigningKey::generate(&mut csprng);
 
         Self {
             secret,
-            secret_crypto_box: OnceCell::default(),
-        }
-    }
-
-    /// Serialise this key to OpenSSH format.
-    pub fn to_openssh(&self) -> ssh_key::Result<zeroize::Zeroizing<String>> {
```

Comment:
These are useful. Can we have this code somewhere, maybe in examples?

Reply:
I would put this into the `load_key` function, which uses this, when we update iroh-blobs